doc/doxygen/Makefile

```diff
@@ -18,6 +18,8 @@ doc: $(DOCUMENTATION_FORMAT)
 .PHONY: html
 html: src/changelog.md
 	( cat riot.doxyfile ; echo "GENERATE_HTML = yes" ) | doxygen -
+	@echo ""
+	@echo "RIOT documentation successfully generated at $(RIOTBASE)/doc/doxygen/html/index.html"
```

Comment:
Would a terminal emulator be able to make this a clickable link with a `file://` prefix?

Reply:
Probably, nice idea. Updated and squashed.